README.md

```diff
@@ -210,6 +210,8 @@ Advanced usage:
     {batocera}  The ROM's emulator-specific /roms/* directory for Batocera (e.g. "gb")
     {jelos}     The ROM's emulator-specific /roms/* directory for JELOS (e.g. "gb")
     {funkeyos}  The ROM's emulator-specific /Roms* directory for FunKey OS (e.g. "Game Boy")
+    {twmenu}    The ROM's emulator-specific /roms/* directory for FunKey OS (e.g. "gb")
```

Comment:
It's not obvious, but this section of the README gets auto-generated on version bumps. You can revert this and focus on `argumentsParser.ts`.

Reply:
Thanks, yeah... I was wondering if that would fail when writing it but didn't want  to assume which one you'd get to first. I'll probbably try and serialize pull requests in the future... or if you have another idea, feel free to let me know. I have no experience with stuff like this tbh.

Reply:
As far as deconflicting with https://github.com/emmercm/igir/pull/811? There probably isn't a good strategy. Maybe alphabetizing them would reduce the frequency, rather than always appending to the end.

Reply:
It could improve readability, too. I don't know if it would decrease the chance of conflicts really but you might be right for most cases. I think though that conflicts like that are just stuff we have to deal with when working on files simultaneously. That I didn't anticipate that when making both changes myself is probably on me :sweat_smile:. 

I tried to make argumentParser.ts and tokens.md consistent by alphabetizing them both in https://github.com/emmercm/igir/pull/804/commits/342e71fda3a1330863532681430f226783c3ac6c. If that's not what you intended than i'll gladly revert that :).